### PR TITLE
Replace tabs with spaces in the dracut module

### PR DESCRIPTION
**Pull request by pirat89**

### CODE REVIEW

File: repos/system_upgrade/common/actors/commonleappdracutmodules/files/dracut/85sys-upgrade-redhat/do-upgrade.sh [hunk 1]
```diff
@@ -196,8 +196,8 @@ ibdmp() {
 
 bring_up_network() {
     if [ -f /etc/leapp-initram-network-manager ]; then
-	# NOTE(ivasilev) Reverting the change to see if it caused the crash
-	. /lib/dracut/hooks/cmdline/99-nm-config.sh
+    # NOTE(ivasilev) Reverting the change to see if it caused the crash
+    . /lib/dracut/hooks/cmdline/99-nm-config.sh
         . /lib/dracut/hooks/initqueue/settled/99-nm-run.sh
```
Comment (vinzenz) [inline repos/system_upgrade/common/actors/commonleappdracutmodules/files/dracut/85sys-upgrade-redhat/do-upgrade.sh:201]:
```suggestion
            . /lib/dracut/hooks/initqueue/settled/99-nm-run.sh
```

File: repos/system_upgrade/common/actors/commonleappdracutmodules/files/dracut/85sys-upgrade-redhat/do-upgrade.sh [hunk 2]
```diff
@@ -196,8 +196,8 @@ ibdmp() {
 
 bring_up_network() {
     if [ -f /etc/leapp-initram-network-manager ]; then
-	# NOTE(ivasilev) Reverting the change to see if it caused the crash
-	. /lib/dracut/hooks/cmdline/99-nm-config.sh
+    # NOTE(ivasilev) Reverting the change to see if it caused the crash
```
Comment (vinzenz) [inline repos/system_upgrade/common/actors/commonleappdracutmodules/files/dracut/85sys-upgrade-redhat/do-upgrade.sh:199]:
```suggestion
        # NOTE(ivasilev) Reverting the change to see if it caused the crash
```

Comment (fernflower) [reply repos/system_upgrade/common/actors/commonleappdracutmodules/files/dracut/85sys-upgrade-redhat/do-upgrade.sh:199]: Actually, I believe the comment can be removed as well - iirc it's a left-over of trying to apply pvalena's suggestions in initram network pr https://github.com/oamg/leapp-repository/pull/960/files#diff-bcf4627581aafbb8e8864d36b3e5c6a58399c5814ff0e7ff26589e94f5a998caR197

Comment (pirat89) [reply repos/system_upgrade/common/actors/commonleappdracutmodules/files/dracut/85sys-upgrade-redhat/do-upgrade.sh:199]: oki. dropping

File: repos/system_upgrade/common/actors/commonleappdracutmodules/files/dracut/85sys-upgrade-redhat/do-upgrade.sh (hunk 1 above)
Comment (pirat89) [reply repos/system_upgrade/common/actors/commonleappdracutmodules/files/dracut/85sys-upgrade-redhat/do-upgrade.sh:201]: lol :D how I chieved this wrong replacement in such a simple change :-D *facepalm* :dagger:  :jp:

**Review by vinzenz — APPROVED** [review]

**Pull request merged**